### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by carreraprogrammer**
This branch was created to:

- [x] Initialize React app.
- [x]  Set reusable components. Create a directory for your components.
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation

### CODE REVIEW

File: src/Pages/Categories.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>Categories</h1>
+);
+
```
Comment (mwanawabangona) [inline src/Pages/Categories.js]: - [ ] `required`
- [ ] Great job so far 👏 
- [ ] however `Category` Should display a button "Check status" .
- [ ] Please fix it 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @carreraprogrammer ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Pages/Categories.js (hunk 1 above)
Comment (carreraprogrammer) [reply src/Pages/Categories.js]: Thanks for the review, I'll do all the changes

**Review by danielufeli — APPROVED** [review]
Hi @carreraprogrammer ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ChopApproveCopApproveGIF

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ate_comments_counter`

**Pull request merged**